.erb-lint.yml

```diff
@@ -313,7 +313,6 @@ linters:
           - check-radio-collection
           - check-radio-group
           - checkboxes
-          - choose-template-preview
```

Comment:
I see that this is used in a javascript file. How did you detected this one? A failing spec or manually? 

We should also start migrating these kind of JS selectors to the new pattern to be consistent (i.e. `data-choose-template-preview` or whatever)

Reply:
actually, it was a single occurrence, and i have manually searched through code.

Reply:
I may not be clear, but what I tried to say is: 

According to [our documentation](https://docs.decidim.org/en/develop/develop/guide_conventions.html#_javascript), we should try to 

> Do not use CSS classes to bind JavaScript selectors or Ruby specs. Try to use an id (with the js- preffix) or a data-attribute as selector, in order to shrink the couplish between changing technologies.
 
So, this CSS class is only being used for a JS selector in `./decidim-templates/app/views/decidim/templates/admin/questionnaire_templates/preview.js.erb`. Can we change this to a `data-` attribute? 

How this would look like: 

1. Adding it again to the erb-lint.yml file
2. Changing it from the HTML in `./decidim-templates/app/views/decidim/templates/admin/questionnaire_templates/_choose.html.erb` to
```html
<div class="row column" data-choose-template-preview></div>
```
3. Changing it from the other file (`./decidim-templates/app/views/decidim/templates/admin/questionnaire_templates/preview.js.erb`)
```javascript
var $preview = $("div[data-choose-template-preview]");
```

Reply:
addressed in [07de077](https://github.com/decidim/decidim/pull/12081/commits/07de077d4046e971ffdadda7be666b26f86f2dda)